test/ProjectService.Business.UnitTests/Commands/GetUserProjectsCommandTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using LT.DigitalOffice.ProjectService.Business.Commands;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Mappers.ResponsesMappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Responses;
+using LT.DigitalOffice.UnitTestKernel;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace ProjectService.Business.UnitTests.Commands
+{
+    public class GetUserProjectsCommandTests
+    {
+        private GetUserProjectsCommand command;
+        private Mock<IProjectRepository> repositoryMock;
+        private Mock<IProjectResponseMapper> mapperMock;
+
+        private Guid userId;
+
+        private DbProject project1;
+        private DbProject project2;
+        private List<DbProject> projectsEnum;
+
+        private ProjectResponse response1;
+        private ProjectResponse response2;
+
+        [SetUp]
```

Comment:
use OneTimeSetUp

Reply:
In this case, I am breaking several tests in this direction.

Reply:
in the command nothing breaking

Reply:
when used OneTimeSetUp